### ISSUE TITLE
The GloBI snapshop seems to have fewer interactions than the stable version

### ISSUE THREAD

**Issue by qgroom** [globalbioticinteractions/globalbioticinteractions#613]
Snapshot tab separated version 7018465 interactions.tsv
Stable tab separated version 7822671 interactions2.tsv

Am I missing something? Shouldn't the snapshot be bigger?

**Comment by jhpoelen** [comment]
Thanks for sharing your observation. About the reduction in interaction counts. A recent discussion/review #602 highlighted some indexed datasets . Some duplicate datasets are now ignored and this explains the reduction in indexed interactions. While the provenance of these indexed records clearly indicate where the records came from (e.g., two identical datasets at different locations), an effort is made to hide these obvious copies from appearing in the indexed dataset produced via https://globalbioticinteractions.org . Note that users of [Elton](https://github.com/globalbioticinteractions/elton) can still access any dataset they'd like.

In other words, the number of indexed interactions is necessarily expected to grow continuously. Also recent curation efforts are consistent with the reduction in interaction records that you shared.

Please let me know if you have additional questions/comments/ideas, otherwise please close this issue.

**Comment by qgroom** [comment]
Thanks for the explaination!
